Source/Core/UICommon/ResourcePack/Manager.cpp

```diff
@@ -0,0 +1,171 @@
+// Copyright 2018 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "UICommon/ResourcePack/Manager.h"
+
+#include "Common/FileSearch.h"
+#include "Common/FileUtil.h"
+#include "Common/IniFile.h"
+
+#include <algorithm>
+
+namespace ResourcePack
+{
+std::vector<ResourcePack> packs;
```

Comment:
this and `packs_path` should be placed in an anonymous namespace or be static (or just not be global state at all).